nifi-commons/nifi-utils/src/main/java/org/apache/nifi/processor/util/StandardValidators.java

```diff
@@ -667,6 +667,46 @@ public ValidationResult validate(final String subject, final String input, final
         };
     }
 
+    public static Validator createRegexMatchingValidatorWithEL(final Pattern pattern, final String validationMessage) {
+        return new Validator() {
+            @Override
+            public ValidationResult validate(final String subject, final String input, final ValidationContext context) {
+                final PropertyValue prefixProperty = context.newPropertyValue(input);
+                final String prefix;
```

Comment:
@timeabarna I retested the processor with a simple flow and it works properly.

A minor question: What does "prefix" mean in this context?

Reply:
@turcsanyip thanks for your comment, renamed it to propertyValue